### PR TITLE
[DOC] fixed README to notify users who installed via pip of difference in docs

### LINKED ISSUE #120: [BUG] Breaking Changes to docs not notified to users who `pip install`
**Describe the bug**
The docs for the master branch do not notify the users who install the program via `pip install` that the documentation for the latest release is not up to date. This issue would fix that. This is necessary as the latest release features breaking changes.

### PR DESCRIPTION
**Pull request by anirudhRowjee**
# Description

This PR is a documentation change WRT #120 .

Fixes #120 

## Type of change

Please delete options that are not relevant.

# Checklist:

- [x] I have read the [contribution guidelines](https://browser-history.readthedocs.io/en/latest/contributing.html) and followed it as far as possible. 
- [ ] I have performed a self-review of my own code (if applicable)
- [ ] I have commented my code, particularly in hard-to-understand areas (if applicable)
- [x] I have made corresponding changes to the documentation (if applicable)
- [ ] My changes generate no new warnings 
- [ ] I have enabled the pre-commit hook and it's not detecting any issue.
- [ ] Any dependent and pending changes have been merged and published

### CODE REVIEW

**Comment by Samyak2** [comment]
Thanks for the PR!

The statement is not correct though. The incompatibilities are because the last release was done months ago, around September 25. There have been many breaking changes made since then (esp. due to Hacktoberfest PRs) and consequently the PyPI version is very out of date compared to master. So the docs for users installing through pip can be found on the `release` branch, by selecting `stable` in the documentation page or on the PyPI page.

**Comment by anirudhRowjee** [comment]
How can I change my statement so that it's more accurate? I'd appreciate any guidance with respect to that :D

File: README.md [hunk 1]
```diff
@@ -10,6 +10,12 @@
 ``browser-history`` is a simple, zero-dependencies, developer-friendly python
 package to retrieve (almost) any browser's history on (almost) any platform.
 
+## NOTE - Docs need to be updated for latest release, as it features breaking changes.
```
Comment (Samyak2) [inline README.md:13]:
```suggestion
## NOTE - The features and quickstart here are for the master branch, which has many breaking changes.
```

File: README.md [hunk 2]
```diff
@@ -10,6 +10,12 @@
...
+
+Some of the code in this README might not work if you have just `pip install`ed the project. Latest docs can be found in the `release` branch.
```
Comment (Samyak2) [inline README.md:15]:
```suggestion
Some of the code in this README might not work if you have just `pip install`ed the project. The docs for the latest release can be found [here](https://browser-history.readthedocs.io/en/stable/).
```

**Review by Samyak2 — COMMENTED** [review]
I've added my suggestions in the review, feel free to change the wording if you think it can be improved xD.

I have one concern with this. When making a release, we'll have remove this part or it will end up on the [PyPI page](https://pypi.org/project/browser-history/) which wouldn't look good. Also, I hadn't noticed it before, but the links on the release branch all point to the `latest` docs which are made from master.
Instead of manually updating all this, it would be nice to be able to automate it. Maybe a custom script?

(this should probably be a separate issue lol, but let me know if you know of any good solutions for this)

**Review by Samyak2 — APPROVED** [review]
LGTM

**Pull request merged**